theme/src/resources/templates/category_pages/liferay_commerce.hbs

```diff
@@ -34,7 +34,7 @@
 			alertChildren = {
 				children: '{{dc 'commerce-japanese_banner_description'}}',
 				linkText: '{{dc 'commerce-japanese_banner_cta'}}',
-  				url: "/hc/ja/categories/360002946411"
+  				url: "https://www.liferay.com/documents/232187258/239574733/Liferay+Learn+Commerce.pdf/a454b269-d196-dc73-c6c1-6782ba66e2ae?t=1586391469883"
```

Comment:
Hey @cole-liferay 
can you remove the uuid and timestamp at the end of the url?